validator/bin/3d-tiles-validator.js

```diff
@@ -1,2 +1,15 @@
 #!/usr/bin/env node
 'use strict';
+
+var argv = require('yargs').argv;
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+
+var readTileset = require('../lib/readTileset');
+var validateTileset = require('../lib/validateTileset');
+
+var filePath = argv._[0];
+var response = validateTileset(readTileset(filePath));
```

Comment:
This still needs to be fixed, using `.then`

Make sure to check that it works with a command like: `node bin/3d-tiles-validator.js specs/data/Tileset/tileset.json`